experimental/BasisLieHighestWeight/src/UserFunctions.jl

```diff
@@ -125,8 +125,18 @@ function basis_lie_highest_weight(
   type::Symbol, rank::Int, highest_weight::Vector{Int}; monomial_ordering::Symbol=:degrevlex
 )
   L = lie_algebra(QQ, type, rank)
+  M = SimpleModuleData(L, highest_weight)
   operators = operators_asc_height(L)
-  return basis_lie_highest_weight_compute(L, highest_weight, operators, monomial_ordering)
+  return basis_lie_highest_weight_compute(M, operators, monomial_ordering)
+end
+
+function basis_lie_highest_weight_demazure(
```

Comment:
please put the new Demazure related functions to the end of the file, and not in-between the simple modules